backend/routers/slack.router.js

```diff
@@ -24,85 +20,53 @@ if (process.env.NODE_ENV !== "test") {
 }
 
 //Finds Id number of channel
-router.get("/findId", (req, res) => {
-  publishMessage();
-  findEvent();
-  // findProject();
-});
+router.get("/scheduleMessage", findCurrentEvents, scheduleMessage);
 
-//uses Id number to send message to said channel
-router.post("/postMeeting/:id", (req, res) => {
-  publishMessage1();
-});
+async function scheduleMessage(req, res, next) {
```

Comment:
I really like that you are creating a separate function for scheduling a message, but do we need to hardwire function to the api res object? I feel like we can either make this function not be Express Middleware and be only a function that we send the Events to, or maybe review if we can turn this into an Express endpoint and not be Middleware.

Reply:
@nickbeaird Looks like the PR author follows the current development style of the application. Don't forget about the existing legacy code. If you'd like to refactor this please create a followup task that can be done as part of resolving the application technical debt.
@Jonathanko52 skip this comment.

Reply:
@alex-anakin, I respectfully must disagree. The use of Express middleware was only recently added by me, so there has not been any prior use of middleware to stipulate this as a current development style

From the Express docs, middleware is often used for validation of endpoints (https://expressjs.com/en/guide/using-middleware.html). I will restate that this method seems to be a really great method that could be used by a variety of endpoints. I don't think that using middleware is the most sustainable coding practice moving forward, and that pattern of mutating the response object in middleware endpoints can really make for challenging debugging later. I am suggesting that we use this method as either an endpoint (i.e (req, res)), or as a helper method.